src/panoptes/pocs/focuser/birger.py

```diff
@@ -63,10 +63,13 @@ def __init__(self,
                  model='Canon EF-232',
                  initial_position=None,
                  dev_node_pattern='/dev/tty.USA49*.?',
+                 max_command_attempts=5,
                  *args, **kwargs):
         super().__init__(name=name, model=model, *args, **kwargs)
         self.logger.debug('Initialising Birger focuser')
 
+        self.max_command_attempts = max_command_attempts
```

Comment:
I think this line needs to come before calling `super().__init__`. Right now I'm getting this error on creating a focuser:
```
D 01-12 03:59:24.850 (03:59:24 UTC) | panoptes.pocs.camera.camera _create_subcomponent:1039        | Creating the base_class_name='AbstractFocuser' object from dict
D 01-12 03:59:24.857 (03:59:24 UTC) | panoptes.pocs.focuser.focuser __init__:99                    | Focuser created: Birger Focuser on /dev/ttyUSB0
D 01-12 03:59:24.873 (03:59:24 UTC) | panoptes.pocs.focuser.serial _connect:107                    | Established serial connection to Birger Focuser on /dev/ttyUSB0.
E 01-12 03:59:24.879 (03:59:24 UTC) | huntsman.pocs.utils.pyro.cli service:87                      | Pyro huntsmanpi000 shutdown unexpectedly AttributeError("'Focuser' object has no attribute 'max_command_attempts'")
```